### PR TITLE
docs(blog): alexa -> android port tutorial

### PR DESCRIPTION
**Pull request by space-pope**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [x] I have run `npm test` against my changes and tests pass.
- [x] I have added or edited necessary documentation, or no docs changes are needed.

### Description

Adding the Alexa -> Android tutorial and a couple supporting documents and updating the main tutorial TOC to reflect the addition.

I'll have one more commit to add here once I get a minor release of the Android library out and its feature integrated in the demo code, but I wanted to go ahead and post the PR so any other issues could be found in the meantime.

### CODE REVIEW

File: content/blog/porting-the-alexa-minecraft-skill-to-android-using-spokestack/index.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+            break;
+    }
+}
+```
+
+Notice what comes _after_ the NLU does its thing. That's our next step: putting another small piece of Amazon's backend right in the app.
+
+## Recreating Dialog Management
+
+A dialog manager takes the intent and slots from the NLU along with the current context (or "state") of the conversation and decides what sort of response the system should give. In the Alexa SDK, this means that the system consults, [in order](https://developer.amazon.com/en-US/docs/alexa/alexa-skills-kit-sdk-for-java/handle-requests.html#handler-processing-order), a series of [`RequestHandler`s](https://developer.amazon.com/en-US/docs/alexa/alexa-skills-kit-sdk-for-java/develop-your-first-skill.html#implementing-request-handlers) that have been registered to a [`SkillStreamHandler`](https://developer.amazon.com/en-US/docs/alexa/alexa-skills-kit-sdk-for-java/develop-your-first-skill.html#implementing-the-skillstreamhandler).
```
Comment (pdtyreus) [inline content/blog/porting-the-alexa-minecraft-skill-to-android-using-spokestack/index.md]: You might consider putting the 's' outside of the link here as the rendered code + link markup looks a little odd. E.g. [`RequestHandler`](https://developer.amazon.com/en-US/docs/alexa/alexa-skills-kit-sdk-for-java/develop-your-first-skill.html#implementing-request-handlers)s. I guess both ways look a little odd.

Comment (space-pope) [reply content/blog/porting-the-alexa-minecraft-skill-to-android-using-spokestack/index.md]: Yeah, that's the problem with wanting to be both consistent in formatting and faithful to the original name. Having the link only span the name of the actual class makes sense, though, so I can do it that way instead.

**Comment by pdtyreus** [comment]
A few little things:

1. Can you add "This tutorial is part of the Porting a Smart Speaker Voice App to Mobile series." to the top of the Android tutorial? That way people can easily click back to the intro article for context. It's already in the other tutorials.

2. I think we decided to remove the support links at the bottom since it's in the footer already.

3. Is it worth mentioning the idea of limiting the number of reprompts in the reprompt post? I know some platforms auto-reprompt 3 times and then call it quits. Maybe pros and cons of that strategy are worth discussion?

**Comment by space-pope** [comment]
@pdtyreus That all sounds reasonable. Should we have a standard TOC at the top of these posts with identical wording? I ask because the other posts start out with things like "In Part 2...", but it feels like the Android and RN versions don't really fit in in a linear way. They're related to the others, but not in a way that you'd necessarily want to go back and read the iOS instructions before reading the Android ones. I'm thinking something like the following:

> This post is part of the _Porting a Smart Speaker Voice App to Mobile_ series, which discusses how to turn an Alexa skill into a mobile app using Spokestack as a replacement for Amazon's voice services. Other articles in the series can be found at the following links:

And I do mention the reprompt limit in that article, but I can expand on it a bit.

File: content/blog/porting-a-smart-speaker-voice-app-to-mobile-part-3/index.md [hunk 2]
```diff
@@ -21,7 +21,12 @@ If you’ve built a Google Action or Alexa Skill, you’ve already defined a nat
...
 
-To get started with a free NLU model that’s compatible with Spokestack, [create a free account](/login) and upload your existing Alexa or Dialogflow interaction model. Detailed instructions on how to export and upload are in the "Language Understanding" section of your account portal. Spokestack will process your files and send you an email when your custom model is ready. You can then download the model and add it to your iOS or Android native app.
+To get started with a free NLU model that’s compatible with Spokestack, [create a free account](/login) and upload your existing Alexa or Dialogflow interaction model. The following instructions for that process are paraphrased from our [export guide](/docs/Concepts/export):
```
Comment (timmywil) [inline content/blog/porting-a-smart-speaker-voice-app-to-mobile-part-3/index.md]:
```suggestion
To get started with a free NLU model that’s compatible with Spokestack, [create a free account](/create) and upload your existing Alexa or Dialogflow interaction model. The following instructions for that process are paraphrased from our [export guide](/docs/Concepts/export):
```

File: content/blog/porting-the-alexa-minecraft-skill-to-android-using-spokestack/index.md [hunk 3]
```diff
@@ -0,0 +1,323 @@
+---
+title: Porting the Alexa Minecraft Skill to Android Using Spokestack
+date: '2020-06-09'
+description: A tutorial on porting a simple Alexa skill to Android using Spokestack
+author: josh
+tags: Tutorial
```
Comment (timmywil) [inline content/blog/porting-the-alexa-minecraft-skill-to-android-using-spokestack/index.md:6]: Any other tags here?

**Review by timmywil — APPROVED** [review]

File: content/blog/porting-the-alexa-minecraft-skill-to-android-using-spokestack/index.md (hunk 3 above)
Comment (space-pope) [reply content/blog/porting-the-alexa-minecraft-skill-to-android-using-spokestack/index.md:6]: I wasn't planning on any—I didn't want to make the tag space too sparse—but I guess we could break these down by platform. What do you think?

Comment (timmywil) [reply content/blog/porting-the-alexa-minecraft-skill-to-android-using-spokestack/index.md:6]: Just checking. I think it's fine.

**Pull request merged**